funnel/templates/space.html

```diff
@@ -42,6 +42,20 @@
                 {%- if proposal.user != proposal.speaker %}
                   <small>via {{ proposal.user.pickername }}</small>
                 {%- endif %}
+                {%- if g.user and 'edit-space' in g.permissions %}
+                  {% with %}
+                    {% set oustation_speaker = proposal.location | is_outstation_speaker(speaker_locations, space_location) %}
```

Comment:
Ugh. Don't use `set` if it can be avoided, and don't use template filters. This should be a function on the model.